views/play_radio_buttons.erb

```diff
@@ -0,0 +1,10 @@
+Hello <%= @player_name %>, what will you choose?
+<form action="/throw" method="post">
+  <input type="radio" id='rock' name="throw" value='rock'>
+  <label for="rock">Rock</label><br>
+  <input type="radio" id='paper' name="throw" value='paper'>
+  <label for="paper">Paper</label><br>
+  <input type="radio" id='scissors' name="throw" value='scissors'>
+  <label for="scissors">Scissors</label><br>
+  <input type="submit" value="Play">
+</form>
```

Comment:
I like the idea of using a radio button to choose the weapon :) it's a clever way of making sure that player will not choose something the shouldn't, i.e. shovel ? xD